precompiles/dapps-staking/DappsStaking.sol

```diff
@@ -4,39 +4,42 @@ pragma solidity >=0.7.0;
 
 /// Interface to the precompiled contract on Shibuya/Shiden/Astar
 /// Predeployed at the address 0x0000000000000000000000000000000000005001
+/// For better understanding check the source code:
+/// repo: https://github.com/AstarNetwork/astar-frame
+/// code: precompiles/dapps-staking/src
```

Comment:
Actually, the comment was to reference the pallet code - so please change that before the merge.
We have detailed rustdoc there.

Reply:
changed